.github/workflows/ci.yaml

```diff
@@ -199,3 +192,11 @@ jobs:
           cmake .
           make
           ./cpp-tests
+      - name: Setup iOS build
+        if: matrix.os == 'macos-10.15'
+        run: |
+          rustup target add x86_64-apple-ios aarch64-apple-ios
+          cargo install cargo-lipo
+      - name: Build for iOS
+        if: matrix.os == 'macos-10.15'
+        run: cargo lipo --release -p ironoxide-cpp
```

Comment:
I thought it was clearer to keep them separated. ¯\_(ツ)_/¯ Setting up the build is different from doing the build. Since the outputs of `cargo install` and `cargo lipo` are very similar, and very long, it's hard to tell which one is talking if you have to actually read the logs.

Reply:
I can see it both ways. Normally I would be all for breaking it up, but it feels a little different here since there are 2 conditionals in a row that are the same.
But I see what you're saying and it's probably good as it is.